src/Midgard/CreatePHP/RdfMapperInterface.php

```diff
@@ -112,6 +95,25 @@ public function store(EntityInterface $entity);
     public function getBySubject($subject);
 
     /**
+     * Get the name for the object.
+     *
+     * Ensure the name is transformed into the canonical name string for
+     * the passed object.
+     *
+     * This may include fixing
+     * uppercase, normalizing doctrine proxy class name to original class
+     * name and so on.
+     *
+     * If you do not know what to do, just check if its an object and if so
+     * return get_class, otherwise return the string as passed in.
+     *
+     * @param object $object
+     *
+     * @return string the canonical name of this object
+     */
+    public function objectToName($object);
```

Comment:
this is the one actual BC break. i suggest to bump branch-alias in composer.json (which we seem to have missed to bump to 1.0) to 1.1-dev and add a note for this interface change in the changelog. something like:

* add 1.0 title to current changelog
* above, add 1.1 title
* say "Added ChainRdfMapper and ChainedRdfMapperInterface to allow using more than one mapper in parallel."... and "BC break: If you implemented your own mapper, note that RdfMapperInterface::objectToName was added"...